### ISSUE TITLE
It doesn't work with local files.

### ISSUE THREAD

**Issue by misiurina** [BuilderIO/figma-html#55]
It just doesn't download anything when used with local html files.

**Comment by ameyrathi** [comment]
Facing the same issue

**Comment by steve8708** [comment]
Hey all - you'll want to use the [chrome extension](https://chrome.google.com/webstore/detail/html-to-figma/efjcmgblfpkhbjpkpopkgeomfkokpaim?hl=en-US) for capturing local files

Please feel free to reopen this if that does not work for you

**Comment by Fallibilist** [comment]
@steve8708 I'm using the chrome extension and have the same issue. For local html files opened in the browser the chrome extension simply says "Done" and doesn't download a figma .json file
![image](https://user-images.githubusercontent.com/29155351/190519584-0a7f937d-a617-4489-9413-3898b477037a.png)

**Comment by valioDOTch** [comment]
Same here.
I tested this as I wanted to easily screencap a HTML mail from Outlook.
However it wasn't related to the special format (*.mht) or that I was relying on Edge.
It was just related to the fact, that the extension can't be used with local files; 
possibly a security safeguard that applies to everyone?

I worked around it by using a local http server (I opted for https://www.uwamp.com/ )

**Comment by valioDOTch** [comment]
@steve8708 Normal GitHub users can't reopen this issue BTW; we could only repost it ;-)

**Comment by steve8708** [comment]
ah! thanks @valioDOTch, reopenning

**Comment by steve8708** [comment]
yeah for anyone interested - in the interim before we find a fix here, a local HTTP server should really help. I often use [http-server](https://www.npmjs.com/package/http-server) for this

so if your file is something like `site/page.html`

assuming you have [nodejs](https://nodejs.org/en/) installed, you can just run

```
npx http-server ./site
```

and then that'll host your local file(s) on `http://localhost:8080/page.html` and then you can use the [chrome extension](https://chrome.google.com/webstore/detail/html-to-figma-by-builderi/efjcmgblfpkhbjpkpopkgeomfkokpaim?hl=en-US) against that URL

hope this helps!
